src/assets/scripts/airport/StandardRoute/RouteSegmentModel.js

```diff
@@ -1,30 +1,79 @@
+import _forEach from 'lodash/forEach';
 import _map from 'lodash/map';
+import _isArray from 'lodash/isArray';
 import _uniqId from 'lodash/uniqueId';
 import StandardRouteWaypointModel from './StandardRouteWaypointModel';
 
 /**
+ * Provides an interface for dealing with a list of `StandardRouteWaypointModel`s that make up a given route segment.
+ *
  * @class RouteSegmentModel
  */
 export default class RouteSegmentModel {
     /**
+     * segmentWaypoints should come in a similar shape to:
+     * - ["_NAPSE068", "NAPSE", ["RIOOS", "A130+"], "COMPS"]
+     *
      * @constructor
-     * @param name {string}
-     * @param segmentWaypoints {array}
+     * @param name {string}  Icao of particular waypoint
```

Comment:
No, this does matter, and I agree icao does not make sense here.  Naming is one of the hardest parts of programming, thus things should be named accurately.